plugins/slack/alerta_slack.py

```diff
@@ -153,33 +147,36 @@ def _slack_prepare_payload(self, alert, status=None, text=None):
 
         return payload
 
-    def post_receive(self, alert):
+    def post_receive(self, alert, **kwargs):
+        SLACK_WEBHOOK_URL = self.get_config('SLACK_WEBHOOK_URL', type=str, **kwargs)
 
         if alert.repeat:
             return
 
         try:
-            payload = self._slack_prepare_payload(alert)
+            payload = self._slack_prepare_payload(alert, **kwargs)
 
             LOG.debug('Slack payload: %s', payload)
         except Exception as e:
             LOG.error('Exception formatting payload: %s\n%s' % (e, traceback.format_exc()))
             return
 
         try:
-            r = requests.post(SLACK_WEBHOOK_URL,
+             r = requests.post(SLACK_WEBHOOK_URL,
```

Comment:
This leading space needs to be removed. Thanks.

Reply:
Done